extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## [Unreleased]
 
+- Add the AST Viewer to inspect the QL AST of a source file in a database. Currently, only available for C/C++ sources.
 - Fix error with choosing qlpack search path.
+- Add the AST Viewer to inspect the QL AST of a source file in a database. Currently, only available for C/C++ sources.
```

Comment:
This line exists twice

Reply:
Thanks for pointing that out. Must have been a bad merge.